tests/test_plotter.py

```diff
@@ -90,3 +90,42 @@ def test_prepare_smooth_shading_not_poly(hexbeam):
     )
 
     assert np.allclose(mesh[scalars_name], expected_mesh[scalars_name])
+
+
+def test_get_datasets(sphere, hexbeam):
+    """Test edge cases for smooth shading"""
```

Comment:
There doesn't seem to be any smooth shading in this test. Nor obvious edge caseness.

Reply:
Quite right.